examples/storage/couchdb/images/init_couch/configer_couch.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+fqdn=$(hostname --fqdn)
+
+cp /core-config/default.ini $INSTALL_LOCATION/etc/default.ini
+cp /core-config/local.ini $INSTALL_LOCATION/etc/local.ini
+cp /core-config/vm.args $INSTALL_LOCATION/etc/vm.args
+mkdir $INSTALL_LOCATION/etc/local.d
+mkdir $INSTALL_LOCATION/etc/default.d
+sed -i $INSTALL_LOCATION/etc/vm.args -e "s/^\-name db/\-name db@$fqdn/"
```

Comment:
This script is run before the the couchdb continer is created, it takes all of the files in the config map and pops them the etc folder.  Ow neat I did not know that Erlang would just use the FQDN if unset.  Sure thing I shall change up the name to couchdb